### PR TITLE
Changes proposed by Sandi implemented, bug in progressbar fixed

### PR DESCRIPTION
**Pull request by rohands**
Progressbar was not visible in the splashscreen in API 21+ devices, fixed this.
Also, Sandi proposed to change the word **Comrades** everywhere in the app.

### CODE REVIEW

**Comment by sandarumk** [comment]
Do not put commit messages like "Changes proposed by Sandi', mention the change such as "Comrade number replaced with Emergency Contacts". Following are some of the good articles I found online regarding this. Please read them.  There are a lot of articles regarding this. You can find some of them and read them too.
http://chris.beams.io/posts/git-commit/
https://wiki.openstack.org/wiki/GitCommitMessages
http://stackoverflow.com/questions/15324900/standard-to-follow-when-writing-git-commit-messages

File: app/src/main/java/com/peacecorps/pcsa/SplashScreenActivity.java [hunk 1]
```diff
@@ -4,6 +4,7 @@
 import android.content.Intent;
 import android.os.Bundle;
 import android.os.Handler;
+import android.util.Log;
```
Comment (sandarumk) [inline app/src/main/java/com/peacecorps/pcsa/SplashScreenActivity.java]: Have you used any logging in this class? Where is this being used in here?

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -91,7 +91,7 @@
     <string name="slide_three_bottom">Select the message you want to send from here!</string>
     <string name="slide_four_head">The message will be sent to 6 Comrades for immediate intervention</string>
```
Comment (sandarumk) [inline app/src/main/res/values/strings.xml]: Shouldn't the text "Comrade" in these change too?

Comment (rohands) [inline app/src/main/res/values/strings.xml]: Yes, there are more occurrences. Will change everything

File: app/src/main/java/com/peacecorps/pcsa/SplashScreenActivity.java (hunk 1 above)
Comment (rohands) [inline app/src/main/java/com/peacecorps/pcsa/SplashScreenActivity.java]: Had used for testing purpose, will remove it.

**Comment by rohands** [comment]
Noted

**Comment by rohands** [comment]
@sandarumk 
I have made the required changes.

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -105,22 +105,22 @@
     <string name="need_to_talk_message">I need to talk. Message sent through First Aide\'s Circle of Trust</string>
     <string name="msg_sent">Message sent</string>
     <string name="ok">OK</string>
-    <string name="message_failed">Message Sending Failed for Comrade </string>
+    <string name="message_failed">Message Sending Failed for Contact </string>
```
Comment (sandarumk) [inline app/src/main/res/values/strings.xml]: Contact should be 'contact' . Sorry about not noticing that with the "Comrades". Do not change the capitalized Emergency Contacts. Change all the other locations accordingly. If you have any doubt ask here or in the mailing list.

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -105,22 +105,22 @@
...
+    <string name="confirmation_message2">registered Contacts.</string>
+    <string name="confirmation_message3">registered Contact.</string>
+    <string name="no_comrade_title">No Contacts Registered</string>
+    <string name="no_comrade_msg">Failed to send message since no Contacts have been registered</string>
     <string name="add_comrades_save">Save</string>
     <string name="message_options">Message Template</string>
 
     <!--Trustees-->
-    <string name="updated_phone_numbers">"Comrades' phone numbers have been updated"</string>
+    <string name="updated_phone_numbers">"Your Emergency Contacts' phone numbers have been updated"</string>
```
Comment (rohands) [inline app/src/main/res/values/strings.xml]: "Emergency Contacts" should remain capitalised here too?

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -105,22 +105,22 @@
     <string name="need_to_talk_message">I need to talk. Message sent through First Aide\'s Circle of Trust</string>
     <string name="msg_sent">Message sent</string>
     <string name="ok">OK</string>
-    <string name="message_failed">Message Sending Failed for Comrade </string>
+    <string name="message_failed">Message Sending Failed for Contact </string>
     <string name="no_phone_number">No phone number found</string>
     <string name="choose_number">Choose a number</string>
-    <string name="circle_of_trust_slide_two">When you are feeling insecure or need comrades\' attention.\nPress the button</string>
+    <string name="circle_of_trust_slide_two">When you are feeling insecure or need your emergenct contacts\' attention.\nPress the button</string>
```
Comment (sandarumk) [inline app/src/main/res/values/strings.xml]: *emergency contacts

File: app/src/main/res/values/strings.xml [hunk 6]
```diff
@@ -105,22 +105,22 @@
...
     <string name="confirmation_message1">This message is being sent to</string>
-    <string name="confirmation_message2">registered Comrades.</string>
-    <string name="confirmation_message3">registered Comrade.</string>
-    <string name="no_comrade_title">No Comrades Registered</string>
-    <string name="no_comrade_msg">Failed to send message since no Comrades have been registered</string>
+    <string name="confirmation_message2">registered Contacts.</string>
+    <string name="confirmation_message3">registered Contact.</string>
+    <string name="no_comrade_title">No Contacts Registered</string>
```
Comment (sandarumk) [inline app/src/main/res/values/strings.xml]: This is okay

File: app/src/main/res/values/strings.xml (hunk 4 above)
Comment (sandarumk) [inline app/src/main/res/values/strings.xml]: I am having second thoughts about that. Better to confirm with Sandi and Patrick. For now let's not capitalize that.  After you make the changes here, can you make a list of strings where you have doubts and let's confirm it with Sandi and Patrick.

Comment (rohands) [inline app/src/main/res/values/strings.xml]: Sure

**Pull request merged**